src/BotPlutusInterface/Effects.hs

```diff
@@ -318,7 +318,7 @@ adaToCApiValue = CApi.lovelaceToValue . CApi.Lovelace . Ada.getLovelace
 
 addValue :: CApi.Value -> Ledger.TxOut -> Ledger.TxOut
 addValue v' (Ledger.TxOut (CApi.TxOut addr (CApi.TxOutValue era v) datum rScript)) = Ledger.TxOut $ CApi.TxOut addr (CApi.TxOutValue era $ v <> v') datum rScript
-addValue _ (Ledger.TxOut (CApi.TxOut _ (CApi.TxOutAdaOnly eraProof _) _ _)) = case eraProof of -- Formatter removes {}?
+addValue _ (Ledger.TxOut (CApi.TxOut _ (CApi.TxOutAdaOnly eraProof _) _ _)) = case eraProof of {} -- Formatter removes {}?
```

Comment:
This comment can go if the formatter no longer does this - do make surre you've run `make format` though